apps/jobs/views.py

```diff
@@ -1589,7 +1597,15 @@ def update_partially_evaluated_submission(request, challenge_pk):
         submission_pk = request.data.get("submission")
         submission_status = request.data.get("submission_status", "").lower()
         job_name = request.data.get("job_name", "").lower()
-        submission = get_submission_model(submission_pk)
+        try:
+            submission = get_submission_model(submission_pk)
+        except NotFound:
```

Comment:
same here

Reply:
@burnerlee @Ram81 Why are we using `get_submission_model` in a `try/catch` block? IIRC, `get_submission_model` is derived from `get_model_object` and it automatically catches the exception here -- https://github.com/Cloud-CV/EvalAI/blob/5698781b99e3fcea100430a2ef85322c0f0545db/apps/base/utils.py#L70.
Am I missing something?